pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -101,9 +101,37 @@ func (m SQSMonitor) processSQSMessage(message *sqs.Message) (*EventBridgeEvent,
 	event := EventBridgeEvent{}
 	err := json.Unmarshal([]byte(*message.Body), &event)
 
+	if err != nil {
+		return &event, err
+	}
+
+	if len(event.DetailType) == 0 {
+		log.Debug().Msg("processing message from source other than EventBridge")
```

Comment:
Do we know for sure that this event is from ASG, or could it be something else? If the former, can we make the debug message more specific? If the latter, does the function `processLifecycleEventFromASG()` need a more generic name?